### PR TITLE
Host/PM - UI: Create account page

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/24467

*Description of changes:*
- account creation screen/flow
- removed /strata-hotel route prefixes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-base-web/app/components/connect/contact/Bcros.vue
Comment (deetz99) [inline strr-base-web/app/components/connect/contact/Bcros.vue:1]: added icons and blue text colour

File: strr-base-web/app/components/modal/Base.vue
Comment (deetz99) [inline strr-base-web/app/components/modal/Base.vue:1]: hide error icon option

File: strr-host-pm-web/app/app.config.ts
Comment (deetz99) [inline strr-host-pm-web/app/app.config.ts:1]: only allow bcsc for host/pm

File: strr-host-pm-web/app/middleware/choose-account-page.ts
Comment (deetz99) [inline strr-host-pm-web/app/middleware/choose-account-page.ts:1]: choose account page redirects to create account page if no accounts found

File: strr-host-pm-web/app/pages/application.vue
Comment (deetz99) [inline strr-host-pm-web/app/pages/application.vue:1]: nuxt will create this as just '/application' now

**Review by thorwolpert — APPROVED** [review]
looks good, do we not have a tracking issue to add this?

**Comment by deetz99** [comment]
> looks good, do we not have a tracking issue to add this?

No but I can add one

**Pull request merged**